AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -416,7 +454,7 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
 
   ASCellNode *node = [_dataController nodeAtIndexPath:indexPath];
   
-  [_rangeController configureContentView:cell.contentView forCellNode:node];
+  [_rangeController configureContentView:cell.contentView forNode:node];
```

Comment:
I'm a little worried about these new ones not being cells.  It means they'll miss out on the fairly useful / important property .neverShowPlaceholders, for one thing; it will also defeat the universality of an override point that ASCellNode currently provides for content in collection views.

The name is definitely not ideal (cell), but from a functionality perspective, would there be a downside to using cell nodes for the supplementary nodes?  Perhaps the best solution is to create a subclass of ASCellNode, for the primary (sole?) purpose of giving it a better name for this use case (from the developer-facing part of the API, so it is not confusing to people to see "cell" for supplementary elements that are perhaps smaller and secondary to real cells; clearly the naming is less critical for this internal stuff)